lib/bot.js

```diff
@@ -91,6 +91,16 @@ class Bot {
       ...this.ircOptions
     };
 
+    // default encoding to UTF-8 so messages to Discord aren't corrupted
+    if (!Object.prototype.hasOwnProperty.call(ircOptions, 'encoding')) {
```

Comment:
Hm, why did you use `Object.prototype.hasOwnProperty` instead of directly on `ircOptions`?

Reply:
The linter complained – I said in [this comment](https://github.com/reactiflux/discord-irc/pull/315#discussion_r142007077).

Reply:
ah lol, sorry about that :)